ploneorg/core/profiles/default/workflows/foundation_member_workflow/definition.xml

```diff
@@ -10,8 +10,34 @@
  <state state_id="approved" title="Approved member">
   <exit-transition transition_id="reject"/>
   <exit-transition transition_id="resign"/>
+  <exit-transition transition_id="suspend"/>
+  <exit-transition transition_id="retire"/>
```

Comment:
I'm curious what the semantic differences are between reject, resign, suspend, retire. Is suspend like 'reject' in the simple pub wf?

Reply:
I was trying to avoid removing anything that was already there, but 'reject' shouldn't be needed here. It puts the member back into the initial state, which we shouldn't need after they've been approved

Reply:
Retire is an action for the admins, Resign is for the member. But they both put the member in the emeritus state, and could be combined